# HG changeset patch
# User Michael Goffioul <michael.goffioul@gmail.com>
# Date 1374156241 14400
#      Thu Jul 18 10:04:01 2013 -0400
# Node ID 477ed3b5549710855ae34910ac49a56eb11c4ae6
# Parent  5069b6c6c062469f9d773a3ad287b282b2b6d77b
QWinTerminalImpl.cpp: Don't define _WIN32_WINNT unconditionally.

* libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp: Don't define
_WIN32_WINNT unconditionally.

diff --git a/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp b/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
--- a/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
+++ b/libgui/qterminal/libqterminal/win32/QWinTerminalImpl.cpp
@@ -35,17 +35,19 @@ along with Foobar.  If not, see <http://
 #include <QCursor>
 #include <QMessageBox>
 
 #include <fcntl.h>
 #include <io.h>
 #include <stdio.h>
 #include <stdarg.h>
 #define WIN32_LEAN_AND_MEAN
-#define _WIN32_WINNT 0x0500 
+#if ! defined (_WIN32_WINNT) && ! defined (NTDDI_VERSION)
+#define _WIN32_WINNT 0x0500
+#endif
 #include <windows.h>
 #include <cstring>
 #include <csignal>
 #include <limits>
 
 #include "QWinTerminalImpl.h"
 #include "QTerminalColors.h"
 
